File: CopassAppWBtn/app/src/main/java/com/example/copassappwbtn/MainActivity.java
package com.example.copassappwbtn;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {
    Button btnOpenCopass;
    Intent intentCompassActivity;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnOpenCopass = findViewById(R.id.btnOpenCopassView);

        btnOpenCopass.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.btnOpenCopassView:
                intentCompassActivity = new Intent(this, CompassActivity.class);
                startActivity(intentCompassActivity);
            default:
                break;
        }
    }
}

File: HelloKotlin/app/src/main/java/com/example/hellokotlin/MainActivity.kt
package com.example.hellokotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val btnJava = findViewById<Button>(R.id.btnjava)
        val btnKotlin = findViewById<Button>(R.id.btnkotlin)
        val btnVarJava = findViewById<Button>(R.id.btnVarjava)
        val btnVarKotlin = findViewById<Button>(R.id.btnVarKotlin)

        btnJava.setOnClickListener {
            //Toast.makeText(this@MainActivity,"HELLO Kotlin",Toast.LENGTH_LONG).show()
            startActivity(Intent(this@MainActivity,BmiJavaActivity::class.java))
        }
        btnKotlin.setOnClickListener {
            startActivity(Intent(this@MainActivity,BmiKotlinActivity::class.java))

        }
        btnVarJava.setOnClickListener {
            startActivity(Intent(this@MainActivity, VariableJavaActivity::class.java))
        }
        btnVarKotlin.setOnClickListener {
            startActivity(Intent(this@MainActivity, VariableKotlinActivity::class.java))
        }


        }


    }